src/GameLoop.tsx

```diff
@@ -533,15 +518,71 @@ export function GameLoop() {
         playerPhysics.hasMoved = false;
     }
 
+    const AppStyle = {
+        display: "flex",
+        border: "solid green 4px",
+        flexWrap: "wrap",
+        flexDirection: "row",
+    };
+
     return (
-        <BoardStyled>
-            <PlayerComponent
-                isVisible={isPlayerVisible}
-                adjustedCells={playerPhysics.adjustedCells}
-            />
-            <BoardComponent
-                boardCellMatrix={boardPhysics.boardCellMatrix}
-            />
-        </BoardStyled>
+        <div style={AppStyle}>
+            <BoardStyled>
+                <PlayerComponent
+                    isVisible={isPlayerVisible}
+                    adjustedCells={playerPhysics.adjustedCells}
+                />
+                <BoardComponent
+                    boardCellMatrix={boardPhysics.boardCellMatrix}
+                />
+            </BoardStyled>
+            <WordList displayedWords={matchedWords} />
+        </div>
     );
 }
+
+const WordList = React.memo(
+    ({ displayedWords }: { displayedWords: string[] }) => {
+        const wordStyle = {
+            background: "yellow",
+        };
+
+        const outerStyle = {
+            display: "flex",
+            flexDirection: "column",
+        };
+
+        const scrollBoxStyle = {
+            flex: "auto",
+            overflowY: "auto",
+            height: "0px",
+        };
+
+        console.log(
+            displayedWords.reverse().map((word, i) => (
+                <div key={`word${i}`} style={wordStyle}>{word}</div>
+            )),
+        );
+
+        return (
+            <div style={outerStyle}>
+                <div flex={"none"}>
+                    Matched Words ({displayedWords.length})
+                </div>
+                <article style={scrollBoxStyle}>
+                    <>
+                        {displayedWords.map((word, i) => (
+                            // Invert the key to keep scroll bar at bottom if set to bottom.
+                            <div
+                                key={`word${displayedWords.length - i}`}
```

Comment:
Why do we need `word` as a prefix in the key?

Reply:
Removed `word`